ui/js/controllers/perf/alerts.js

```diff
@@ -285,6 +285,10 @@ perf.controller('AlertsCtrl', [
             }
         };
 
+        $scope.copyTextToClipboard = function(alertSummary) {
+            clipboard.copyText(JSON.stringify(alertSummary));
```

Comment:
We actually want a textual summary here, using the function @crosscent defined here:

https://github.com/mozilla/treeherder/blob/b71fb9d/ui/js/controllers/perf/alerts.js#L15

I would take that function, add it to the alert summary prototype (as e.g. `getTextualSummary`) then write this as:

```js
clipboard.copyText(alertSummary.getTextualSummary());
```
The alert summary prototype is defined around here:

https://github.com/mozilla/treeherder/blob/master/ui/js/models/perf/alerts.js#L113

You'll obviously also want to update the PhBugs class to use this function as well.